Mathlib/RingTheory/Polynomial/Irreducible.lean

```diff
@@ -0,0 +1,146 @@
+/-
+Copyright (c) 2023 Andrew Yang, Patrick Lutz. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Andrew Yang
+-/
+import Mathlib.RingTheory.Norm
+
+/-!
+# Criteria for `X ^ n - C a` to be irreducible
```

Comment:
Splitting out the part about the polynomial seems reasonable, but the file name should be more specific, maybe "Kummer"? I do see "Kummer polynomials" used in the literature to refer to polynomials of the form `X ^ n - a` that are irreducible.